tests/DocumentationTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Support\Str;
+use PHPUnit\Framework\TestCase;
+
+class DocumentationTest extends TestCase
+{
+    public function testIfDocumentationContainsAllPublicFunctions(): void
+    {
+        $functions = $this->getPublicFunctions();
+
+        foreach ($functions as $class => $methods) {
+            $trait = explode("\\", $class);
+            $fileName = Str::kebab(end($trait)) . ".html";
+            $filePath = "docs/elements/$fileName";
+
+            if (!file_exists($filePath)) {
+                if (str_ends_with($filePath, "http-request.html")) {
+                    $filePath = str_replace("http-request.html", "http.html", $filePath);
+                } elseif (str_ends_with($filePath, "http-response.html")) {
+                    $filePath = str_replace("http-response.html", "http.html", $filePath);
+                }
+
+                if (!file_exists($filePath)) {
+                    $this->fail("Documentation file $filePath does not exist.");
+                }
+            }
+
+            $fileContent = file_get_contents($filePath);
+
+            if ($fileContent === false) {
+                $this->fail("Unable to read the file $filePath.");
+            }
+
+            foreach ($methods as $method) {
+                $containsFunction = strpos($fileContent, $method) !== false;
+                $this->assertTrue($containsFunction, "Function $method is not documented in $filePath.");
+            }
+        }
+    }
+
+    protected function getBltTraits(): array
+    {
+        $classes = include "vendor/composer/autoload_classmap.php";
+        $bltTraits = [];
+
+        foreach (array_keys($classes) as $class) {
+            if (str_contains($class, "Blumilk\BLT\Features\Traits")) {
```

Comment:
Maybe https://www.php.net/manual/en/function.str-starts-with.php would be better?

Reply:
Are you going to change that?

Reply:
@JakubKermes

Reply:
Done